spring/spring-data-primitive-types/src/test/java/io/quarkus/ts/spring/data/primitivetypes/di/SpringDiIT.java

```diff
@@ -0,0 +1,106 @@
+package io.quarkus.ts.spring.data.primitivetypes.di;
+
+import static org.junit.jupiter.api.Assertions.assertEquals;
+import static org.junit.jupiter.api.Assertions.assertNotNull;
+
+import java.util.stream.Stream;
+
+import javax.enterprise.inject.spi.CDI;
+
+import org.junit.jupiter.api.Test;
+import org.junit.jupiter.params.ParameterizedTest;
+import org.junit.jupiter.params.provider.Arguments;
+import org.junit.jupiter.params.provider.MethodSource;
+
+import io.quarkus.arc.Arc;
+import io.quarkus.arc.InjectableBean;
+import io.quarkus.test.junit.QuarkusTest;
+
+// Not converted into @QuarkusScenario, because invoking javax.enterprise.inject.spi.CDI / io.quarkus.arc.Arc is not supported.
+// TODO: Workaround by implementing REST endpoints which expose all necessary information.
```

Comment:
To me, it's fine to not convert this method to `@QuarkusScenario`. Keep using `@QuarkusTest` makes sense in scenarios like this.